gsoc.md

```diff
@@ -0,0 +1,24 @@
+# Google Summer of Code
+
+The FEniCS Project has participated in the [Google Summer of Code](https://summerofcode.withgoogle.com) (GSoC) through our fiscal sponsorship agreement with [NumFOCUS](https://numfocus.org/).
+
+## Previous projects
+
+## 2022
+
+- Michele Castriotta (Istituto Italiano di Tecnologia) - [Expanding FEniCSx electromagnetic demos](https://mikics.github.io/gsoc-jupyterbook/submission.html)
+
+## 2019
+
+- Abhinav Gupta (Indian Institute of Technology, Roorkee) - [FEniCS: The mesh workflow](https://computationalmechanics.in/fenics-the-mesh-workflow/)
+- Igor Baratta (Universidade Federal de Minas Gerais)- [Interface to KaHIP partitioner](https://gist.github.com/IgorBaratta/038747f9aff44eb844c0913614bf0c03)
+
+## 2018
+
+- Fabian Löschner (RWTH Aachen University) - [Maximising performance on modern architectures with data-level parallelism](https://flgsoc18.wordpress.com/2018/05/13/excited-for-fenics-and-gsoc/)
+- Igor Baratta (Universidade Federal de Minas Gerais)- [Complex Number support in FEniCS](https://gist.github.com/IgorBaratta/c7ca5252834f2c70efe0d233a3acecb4)
+
+## 2017
+
+- Ivan Yashcuch (Aalto University) -[Develop assembly of finite element forms on quadrilateral and hexahedral meshes in FEniCS](https://nbviewer.org/gist/IvanYashchuk/48bff62b155e31f85de975b24b23e4cb/)
```

Comment:
The spelling of the last name looks dubious. The white space around the hyphen (there should be en dash, in fact) is also random around the file.